Makefile.sdk.mk

```diff
@@ -191,6 +191,9 @@ ifeq ($(host_platform), linux)
 	v8_host_flags := -f make-linux
 	v8_libs_private := " -lm"
 endif
+ifeq ($(host_platform), qnx)
+	v8_host_flags := -f make-qnx -D snapshot=off
```

Comment:
I don't think of it actually does anything. We can probably nuke it. I'll try to check it agian tomorrow.